### ISSUE TITLE
Code does not work for cicularly polarized pulses

### ISSUE THREAD

**Issue by jonyanjiang** [ALaDyn/piccante#8]
Dear developers, I find maybe a bug.
When I use the code to run only two cicularly polarized lasers, it works ok.
But when plasma added, it breaks out, after laser reaching the surface of plasma.
I used the 1.5.2 version.
Can you help me to solve this? Thanks.
Here is the result and the main file.
![circularhhgB_FIELD_000000bin000txt](https://user-images.githubusercontent.com/45502768/67098397-b86a5e80-f1ee-11e9-997e-71a62a974bed.png)
![circularhhgB_FIELD_010000bin000txt](https://user-images.githubusercontent.com/45502768/67098433-c8823e00-f1ee-11e9-9358-ed00f087695c.png)
[main-piccante.txt](https://github.com/ALaDyn/piccante/files/3744134/main-piccante.txt)

**Comment by cenit** [comment]
Thanks for your report. I will investigate it as soon as possible

**Comment by sgatto** [comment]
@cenit , did you reproduce the issue?

**Comment by sgatto** [comment]
@jonyanjiang , can you provide the input file of your simulation?

**Comment by cenit** [comment]
@sgatto no, at least not without an input file or a custom main cpp file.

**Comment by jonyanjiang** [comment]
@sgatto The input file is attached at the end line of the original issue statement. I changed its suffix from .cpp to .txt to meet the github requirement for attached files.
